R/mincolumns.R

```diff
@@ -131,9 +131,9 @@ extremeCols <- function(x, function.name, remove.missing = TRUE, dims)
     } else
     {
         if (isQTable(x) && getDimensionLength(x) > 2 && statisticsPresentInLastDim(x))
```

Comment:
Is the `statisticsPresentInLastDim` check required? The higher dim QTables are already flattened to either 2d tables when no multiple stats are present or a 3d table if multiple stats are present when they pass through the common pre processing function in https://github.com/Displayr/verbs/blob/ea9585c0c7f3416393a9dfbeb14b27c89a5dbaaf/R/utilities.R#L215
So the input should always be at most a 2d array if there are not single stats and a 3d table can only occur for multiple stats?

Reply:
Done